example/ios/Podfile

```diff
@@ -6,6 +6,10 @@ target 'StripeSdkExample' do
   config = use_native_modules!
 
   use_react_native!(:path => config["reactNativePath"])
+pod 'Stripe', :git => 'https://github.com/stripe-ios/stripe-ios.git', :branch => 'private'
+pod 'StripeUICore', :git => 'https://github.com/stripe-ios/stripe-ios.git', :branch => 'private'
+pod 'StripeCore', :git => 'https://github.com/stripe-ios/stripe-ios.git', :branch => 'private'
```

Comment:
Should this podfile work for public users? Not sure about permissions to the private github

Reply:
No this won't work, we'll need to update this back to the original once PaymentSheet is GA'd

Reply:
I just did it this way so I could get the latest build of PaymentSheet so I could test E2E in iOS.